utils/applinks/applinks.go

```diff
@@ -0,0 +1,66 @@
+package applinks
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/imroc/req/v3"
+	"github.com/kickback-app/common/log"
+)
+
+const defaultAppURL = "https://kickbackapp.page.link/89eQ"
+
+var googleCloudAPIKey = os.Getenv("GOOGLE_CLOUD_API_KEY")
+var client = req.C()
+
+func DynamicAppURL(eventID string) string {
+	url := fmt.Sprintf("https://firebasedynamiclinks.googleapis.com/v1/shortLinks?key=%v", googleCloudAPIKey)
+	body := map[string]interface{}{
+		"dynamicLinkInfo": map[string]interface{}{
+			"domainUriPrefix": "https://kickbackapp.page.link",
+			"link":            fmt.Sprintf("https://kickbackapp.io/invited?eventId=%v", eventID),
+			"iosInfo": map[string]interface{}{
+				"iosBundleId": "com.kickbackapp",
+			},
+		},
+	}
+	var result struct {
+		ShortLink   string `json:"shortLink"`
+		PreviewLink string `json:"previewLink"`
+	}
+	var errRes interface{}
+	resp, err := client.R().
```

Comment:
can we use the `common/request` package instead? that way we dont need to import all this stuff -- example usage: https://github.com/kickback-app/common/blob/main/request/request_test.go#L38 as well as a descriptor in this project readme